voorhoede/tudelft-spacefinder
# Remove the studyType data from the project as it is not used anymore

# Changes

Removes the studyType from csv configuration and from the frontend.

# Associated issue

Resolves https://trello.com/c/49Mcq6tY/111-remove-icon-indicating-individual-group-spaces-from-space-cards

# How to test

1. Open preview.
2. Check if the icon is gone from the space cards.

# Checklist
